cmd/registry-disk-v1alpha/entry-point.sh

```diff
@@ -38,6 +38,18 @@ IMAGE_EXTENSION=$(echo $IMAGE_NAME | sed  -n -e 's/^.*\.\(.*\)$/\1/p')
 mkdir -p $COPY_PATH
 echo $IMAGE_NAME | grep -q -e "raw" -e "qcow2"
 if [ $? -ne 0 ]; then
+	qemu-img info $IMAGE_NAME | while read LINE
```

Comment:
Security coding should be error defensive. you are assuming here if you didn't find an issue you are ok.
Is there a way to check it is OK, otherwise always fail.
what happens if qemu-img return some unexpected error/output? qemu-img output changed?
should we check qemu-img returned error code?

Reply:
Maybe I'm missing something....If qemu-img returns error code/unexpected output, there is a problem processing/reading the info. Doesn't it mean that something is wrong with the image?
In other words, is there a way that we still want to convert this image even if qemu returns errors?
Regarding the qemu-img output change... do you mean the format of the fields? Any change in qemu-img has to be backward compatible like other tools (i.e sometimes we parse the output of 'kubectl get pods').
What are you suggesting?

Reply:
@fabiand I wonder what's the advantage of doing this if we are not going to use jq for exmaple...

Reply:
Valid point. I actually wonder if this could be the point to transform the script into a compilable language.

After all I wouldn't block.